src/pages/o/[orgId]/events.tsx

```diff
@@ -4,43 +4,32 @@ import { QueryClient, useQuery } from 'react-query';
 
 function getEvents(orgId) {
     return async () => {
-        try {
-            const cRes = await fetch(`http://localhost:3000/api/orgs/${orgId}/campaigns`);
-            const cData = await cRes.json();
+        const cRes = await fetch(`http://localhost:3000/api/orgs/${orgId}/campaigns`);
+        const cData = await cRes.json();
 
-            let allEventsData = [];
+        let allEventsData = [];
 
-            for (const obj of cData.data) {
-                const eventsRes = await fetch(`http://localhost:3000/api/orgs/${orgId}/campaigns/${obj.id}/actions`);
-                const campaignEvents = await eventsRes.json();
-                allEventsData = allEventsData.concat(campaignEvents.data);
-            }
+        for (const obj of cData.data) {
+            const eventsRes = await fetch(`http://localhost:3000/api/orgs/${orgId}/campaigns/${obj.id}/actions`);
+            const campaignEvents = await eventsRes.json();
+            allEventsData = allEventsData.concat(campaignEvents.data);
+        }
 
+        if (allEventsData) {
```

Comment:
In what scenario could this be falsy?

In what scenario should this method be considered failing?